doc/api/https.md

```diff
@@ -251,7 +252,6 @@ const req = https.request(options, (res) => {
 });
 ```
 
-[`Agent`]: #https_class_https_agent
```

Comment:
I think it would be good to keep this reference. There are other places where the agent is mentioned but not referenced. It would probably be good to make all occurances of the `Agent` a reference instead of removing this.